src/Year.php

```diff
@@ -281,7 +285,13 @@ public function jsonSerialize(): string
      */
     public function toISOString(): string
     {
-        return (string) $this->year;
+        return $this->year < 1000 && $this->year > -1000
```

Comment:
Could you please add the comment:

```php
// This code is optimized for high performance
```

to justify that it’s not using sprintf?

Reply:
Done